scripts/compress_images.py

```diff
@@ -0,0 +1,209 @@
+                            'original_size': original_size,
+                            'new_size': new_size
+                        })
+                else:
+                    logging.info(
+                        'Compressed image > original image'
+                    )
+                    continue
+
+    return result_images
+
+def compress_images_for_workflow(
+        result_images: List[CompressedImageInfo]
+    ) -> None:
+    """Compresses the images using GraphicsMagick."""
+
+    if os.path.exists(OUTPUT_DIR):
+        shutil.rmtree(OUTPUT_DIR)
+    os.makedirs(OUTPUT_DIR, exist_ok=True)
+
+    for images in result_images:
+        file_extension = images['path'].suffix.lower()
+        file_path = images['path']
+
+        rel_path = os.path.relpath(file_path)
+        output_file_path = os.path.join(OUTPUT_DIR, rel_path)
+
+        # Create the directory structure for the output file
+        os.makedirs(os.path.dirname(output_file_path), exist_ok=True)
+
+        if file_extension not in ALL_IMAGES_EXTENSIONS:
+            continue
+
+        with Image.open(file_path):
+            with tempfile.TemporaryDirectory() as tmpdir:
+                temp_compressed = (
+                    pathlib.Path(tmpdir) / f'compressed_{file_path.name}'
+                )
+
+                compression_type = (
+                    'Zip' if file_extension in
+                    IMAGE_EXTENSIONS_SUPPORTING_ZIP_COMPRESSION
+                    else 'LZW'
+                )
+
+                cmd = [
+                    'gm', 'convert',
+                    file_path,
+                    '-strip',
+                    '-compress', compression_type,
+                    output_file_path
+                ]
+
+                result = subprocess.run(
+                    cmd, capture_output=True, text=True, check=False
+                )
+                if result.returncode == 0 and temp_compressed.exists():
+                    with open(temp_compressed, 'rb') as f:
+                        compressed_data = f.read()
+                    # adding compressed images to OUTPUT_DIR.
+                    with open(OUTPUT_DIR, 'wb') as f:
+                        f.write(compressed_data)
+
+                else:
+                    logging.info(
+                        'Compressed image > original image'
+                    )
+                    continue
+
+
+if __name__ == '__main__':  # pragma: no cover
+    # I recommend to do not use this script in local development, as it may
+    # compress images that are not meant to be compressed.
+    # This is only made for CI workflow artifacts.
+    repo_path = pathlib.Path('./assets')
+    compressed_images = get_compressible_images(repo_path)
+
+    if compressed_images:
+        TOTAL_SPACE_SAVED = 0
+        print(len(compressed_images), 'images could be compressed further:\n')
+        for image in compressed_images:
+            print(image['path'])
```

Comment:
```suggestion
            print('    ', image['path'])
```

to make the output easier to read by making it clear that the images are under the "images could be compressed further:" header